src/main/java/org/broadinstitute/hellbender/engine/FeatureManager.java

```diff
@@ -233,6 +233,24 @@ public boolean isEmpty() {
     }
 
     /**
+     * Given a FeatureInput argument field from our tool, returns an iterator to its features starting
+     * from the first one.
+     *
+     * Will throw an exception if the provided FeatureInput did not come from the tool that this
+     * FeatureManager was initialized with, or was not an @Argument-annotated field in the tool
+     * (or parent classes).
```

Comment:
Add a warning in the comment for this method that calling this method a second time on the same `FeatureInput` will invalidate (close) the iterator returned from the first call. (This is because Tribble documentation states that having multiple iterators open simultaneously over the same FeatureReader results in undefined behavior).